### PR TITLE
Adding versioning support to R2

### PR DESCRIPTION
**Pull request by CsatariGergely**
Redoing https://github.com/nephio-project/docs/pull/132 now without merging the main branch to the R2 one. 

Partly fixes https://github.com/nephio-project/nephio/issues/738

### CODE REVIEW

File: content/en/_index.md [hunk 1]
```diff
@@ -27,10 +27,8 @@ and network functions across large scale edge deployments."
 
 {{% blocks/section type="row" %}}
 
-{{% blocks/feature icon="fas fa-rocket" title="R1" %}}
-Welcome to the R1 release of Nephio. Please use the documentation links below to learn more about Nephio R1, or checkout
-our [Learning Nephio with R1](https://wiki.nephio.org/display/HOME/Learning+with+Nephio+R1) series, which includes a
-[demo video](https://youtu.be/mFl71sy2Pdc) and short articles about different aspects of Nephio R1.
+{{% blocks/feature icon="fas fa-rocket" title="R2" %}}
+Welcome to the R2 release of Nephio. Please use the documentation links below to learn more about Nephio R2.
```
Comment (arora-sagar) [inline content/en/_index.md:31]: @CsatariGergely Shouldn't we be in sync with cover title "Welcome to Release 2 of Nephio" ?

**Review by liamfallon — COMMENTED** [review]
OK with me once the comment from @arora-sagar is resolved.

**Comment by CsatariGergely** [comment]
Partially fixes [#738](https://github.com/nephio-project/nephio/issues/738)

File: content/en/_index.md (hunk 1 above)
Comment (arora-sagar) [reply content/en/_index.md]: Okay I am fine with this

**Comment by arora-sagar** [comment]
> Partially fixes [#738](https://github.com/nephio-project/nephio/issues/738)

I suppose we can approve this PR once --> https://github.com/nephio-project/docs/pull/128 is merged

**Pull request merged**